src/components/HeroImage/HeroImage.st.css

```diff
@@ -0,0 +1,45 @@
+:import {
+  -st-from: "../../common/formatters.st";
+  -st-named: color, fallback;
+}
+
+:import {
+  -st-from: "../Image/Image.st.css";
+  -st-default: Image;
+  -st-named: overrideStyleParams as overrideStyleParamsImage, defaultImageOpacity;
+}
+
+/* Defaults */
+:vars {
+  defaultBackgroundColor: color-5;
+}
+
+/* Overrides */
+:vars {
+  /*
+    A background color by a color from the palette for the container
+    @default color-5
+  */
+  BackgroundColor: --overridable;
+  /*
+    An opacity between 0 to 100 for the image
+    @default 100%
+  */
+  ImageOpacity: --overridable;
+}
+
+:vars {
+  backgroundColor: color(fallback(value(BackgroundColor), value(defaultBackgroundColor)));
+  imageOpacity: fallback(value(ImageOpacity), value(defaultImageOpacity));
+}
+
+.root {
+  -st-extends: Image;
```

Comment:
Since you are not extending this Image and you only use the -st-mixin, you don't need the st-extends here.
Moreover, when adding this st-extends you give the ability for our users to target inner classes from the Image component and to override the styling.

For example:
.myHeroImage::anyClassFromTheImageComponent {
   // some styling
}